src/main/java/org/jinstagram/http/Response.java

```diff
@@ -54,7 +54,7 @@ private String parseBodyContents() {
 		for (java.util.Map.Entry<String,List<String>> entry : conn.getHeaderFields().entrySet()) {
 			String key= entry.getKey();
 			List<String> valueList= entry.getValue();
-			if (key!=null && valueList!=null && valueList.size()>0) {
+			if (key!=null && valueList!=null && valueList.isEmpty()) {
```

Comment:
@RobertLara "valueList.size()>0" seems to be the opposite of "valueList.isEmpty()".
```suggestion
			if (key!=null && valueList!=null && !valueList.isEmpty()) {
```